lib/osf-components/addon/components/resources-list/edit-resource/template.hbs

```diff
@@ -0,0 +1,82 @@
+<OsfDialog
+    @onOpen={{perform this.onOpen}}
+    @onClose={{this.onClose}}
+    as |dialog|
+>
+    <dialog.trigger
+        ...attributes
+    >
+        {{yield dialog}}
+    </dialog.trigger>
+    <dialog.heading>
+        {{t 'osf-components.resources-list.edit_resource.heading'}}
+    </dialog.heading>
+    <dialog.main>
+        {{#unless this.shouldShowPreview}}
+            <FormControls
+                @changeset={{this.changeset}} 
+                as |form|
+            >
+                <form.text
+                    @isRequired={{true}}
+                    @valuePath={{'pid'}}
+                    @label={{t 'osf-components.resources-list.edit_resource.doi'}}
+                />
+                <form.select
+                    @isRequired={{true}}
+                    @valuePath={{'resourceType'}}
+                    @label={{t 'osf-components.resources-list.edit_resource.output_type'}}
+                    @options={{this.availableTypes}}
+                    @searchEnabled={{false}}
+                    @renderInPlace={{true}}
+                    as |type|
+                >
+                    {{t (concat 'osf-components.resources-list.edit_resource.' type)}}
+                </form.select>
+                <form.textarea
+                    @isRequired={{true}}
+                    @valuePath={{'description'}}
+                    @label={{t 'osf-components.resources-list.edit_resource.description'}}
+                />
+            </FormControls>
+        {{/unless}}
+        {{#if this.shouldShowPreview}}
```

Comment:
Can we move the above `{{#unless this.shouldShowPreview}}`  to the `{{else}}` block of this condition?